mandatory/1-fix-functions.js

```diff
@@ -21,20 +21,15 @@ function mood() {
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
-
-  if (isBigEnough) {
+  if (num > 10) {
     return "num is greater than 10";
   } else {
     return "num is not big enough";
   }
 }
 
 function get3rdIndex(arr) {
-  let index = 3;
-  let element;
-
-  return element;
+  return arr[3];
```

Comment:
this will return the 4th index, remember in arrays, indexing starts at zero

example:
let arr = [a,b,c,d,e,f]
to get the third index: a is 0 index, b is index 1, c is 2 and d is 3

Hope this helps